### ISSUE TITLE
link for download does not work for Accessibility Guidelines (WCAG) 2.1 Conformance Statement for Power Apps portals

### ISSUE THREAD

**Issue by markgull** [MicrosoftDocs/powerapps-docs#2773]
When clicking on the link to download the 

"W3C Web Content Accessibility Guidelines (WCAG) 2.1 Conformance Statement for Power Apps portals"

it does not work. Instead it gives this error:

<Error>
<Code>ResourceNotFound</Code>
<Message>The specified resource does not exist. RequestId:ac7b2e25-101e-002a-33e4-b65f36000000 Time:2021-10-01T16:53:48.2257194Z</Message>
</Error>

thanks,
Mark

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b8e40af7-ba15-42f8-b9e1-3d1c5ae73285
* Version Independent ID: d63fa884-aa15-e95d-f44b-8ce6eabf7142
* Content: [Accessibility in Power Apps portals - Power Apps](https://docs.microsoft.com/en-us/powerapps/maker/portals/admin/accessibility)
* Content Source: [powerapps-docs/maker/portals/admin/accessibility.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/maker/portals/admin/accessibility.md)
* Service: **powerapps**
* Sub-service: **portals**
* GitHub Login: @neerajnandwana-msft
* Microsoft Alias: **nenandw**

**Comment by nickdoelman** [comment]
This has been resolved.
